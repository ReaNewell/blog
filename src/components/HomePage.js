import React from 'react';

import { Footer } from './Footer';
import { Header } from './Header';
import RecentPosts from './RecentPosts';
import SideBar from './SideBar';

class HomePage extends React.Component {
    render() {
        return (
            <div>
                <Header />
                <img className="homepage__banner" src="#"/>
                <RecentPosts />
                <SideBar />
                <Footer />
            </div>
        )
    }
}

export default HomePage;