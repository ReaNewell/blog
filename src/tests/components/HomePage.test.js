import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../../components/HomePage';

test('should render homepage correctly', () => {
    const wrapper = shallow(<HomePage />);
    expect(wrapper).toMatchSnapshot();
});