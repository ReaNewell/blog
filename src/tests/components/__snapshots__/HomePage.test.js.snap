// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render homepage correctly 1`] = `
<div
  className="homepage"
>
  <TopBar />
  <div
    className="homepage__plantcards"
    onClick={null}
  />
  <PlantForm
    error={false}
    formActive={false}
    handleActivate={[Function]}
    name=""
    onNameChange={[Function]}
    onSubmit={[Function]}
  />
</div>
`;
